README.md

```diff
@@ -12,8 +12,8 @@ Demonstrator tracking chain for accelerators.
 | **Track finding**  | Spacepoint binning     | ✅  | ✅   | ✅   | ⚪      |
 |                    | Seed finding           | ✅  | ✅   | ✅   | ⚪      |
 |                    | Track param estimation | ✅  | ✅   | ✅   | ⚪      |
-|                    | Combinatorial KF       | ⚪  | ⚪   | ⚪   | ⚪      |
-| **Track fitting**  | KF                     | 🟡  | 🟡   | ⚪   | ⚪      |
+|                    | Combinatorial KF       | 🟡  | 🟡   | ⚪   | ⚪      |
+| **Track fitting**  | KF                     | ✅  | ✅   | ✅   | ⚪      |
```

Comment:
Please upgrade the Mermaid graph accordingly, by making the following changes:

The code that currently reads:

```
    %% CPU Kalman filter
    track -.->|Kalman filter| track;
    linkStyle 19 stroke: black;
    %% CUDA kalman filter
    track -.->|Kalman filter| track;
linkStyle 20 stroke: green;
```

Should become:

```
    %% CPU Kalman filter
    track -->|Kalman filter| track;
    linkStyle 19 stroke: black;
    %% CUDA kalman filter
    track -->|Kalman filter| track;
linkStyle 20 stroke: green;
```

(Notice the removal of the dot in the arrow description)

Then add, to the end: 

```
%% SYCL Kalman filter
track -->|Kalman filter| track;
linkStyle 25 stroke: blue;
```